static/script/historian.js

```diff
@@ -121,16 +122,16 @@ define(
              * @returns {Boolean} True if the history stack contains one or more valid return URLs.
              */
             hasHistory: function hasHistory () {
-                var historyMinimumLength = this.hasBroadcastOrigin() ? 2 : 1;
+                var historyMinimumLength = 1;
                 return this._historyArray.length >= historyMinimumLength;
             },
 
             /**
-             * Returns a Boolean to indicate whether the first entry in the history stack is the special 'broadcast' entry.
-             * @returns {Boolean} True if the first entry in the history stack is the special 'broadcast' entry.
+             * Returns a Boolean to indicate whether the URL contains a 'broadcast' query parameter.
+             * @returns {Boolean} True if the URL contains a 'broadcast' query parameter.
              */
             hasBroadcastOrigin: function hasBroadcastOrigin () {
-                return this._historyArray.length > 0 && this._historyArray[this._historyArray.length - 1] === Historian.HISTORY_TOKEN + Historian.BROADCAST_ENTRY;
+                return this._urlParams.length > 0 && this._urlParams[Historian.BROADCAST_ENTRY] === true;
             }
         });
 
```

Comment:
Suggest to remove the BROADCAST_ENTRY and everything to do with history= if we are moving away from this as a concept.
Then update any partner specs we have which i'm sure there were a few years ago.
Need to speak with @aevans-bbc

Reply:
Yep, this was re-factored to remove. I guess we'll need to put some tickets in for the updates to partner specs now?